msi/release.sh

```diff
@@ -185,7 +186,7 @@ cat <<EOF
     </Fragment>
 </Wix>
 EOF
-)>GitComponents.wxs
+) | sed -r -e "s/(<File .*Source=\")(.*(compat-)?(ba)?sh.exe|.*git.*\.exe)([^>]*) \/>/\1\2\" BindPath=\"[BindImagePaths]\5 \/>/" >GitComponents.wxs
```

Comment:
I am so used to reading non-extended regexps in Git's own source code (for portability). So I would prefer to use non-extended regexps here, too, if at all possible... (I think that escaping the parentheses, question marks and the pipe should suffice).